README.md

```diff
@@ -1882,6 +1882,7 @@ A rule that aliases and saves N images into a single `docker save` tarball.
       <td><code>stamp</code></td>
       <td>
         <p><code>Bool; optional</code></p>
+        <p>Deprecated: it is now automatically inferred.</p>
```

Comment:
Thanks for the PR. Could you also update the doc for container_push?

Reply:
Oh, didn't know it's deprecated too. Updated `container_push` too.